searchJob/tests.py

```diff
@@ -0,0 +1,81 @@
+from user_notification.models import JobType, JobCity
+import pytest
+from.models import Company, JobPost
+
+
+@pytest.mark.parametrize('n,expected', [
+    (("bla", "bla", "bla", "bla"), "no relevant jobs for you"),
+    (("SW", None, None, None), "<QuerySet [<JobPost: intel sw for students>]>"),
+    (("SW", "tel aviv", "STUDENT", "intel"), "<QuerySet [<JobPost: intel sw for students>]>"),
+    (("SW", "tel aviv", None, None), "<QuerySet [<JobPost: intel sw for students>]>"),
+    ((None, "tel aviv", None, None), "<QuerySet [<JobPost: intel sw for students>, <JobPost: intel QA>]>"),
+    (("QA", "tel aviv", None, "intel"), "<QuerySet [<JobPost: intel QA>]>"),
+    ((None, None, None, "intel"), "<QuerySet [<JobPost: intel sw for students>, <JobPost: intel QA>]>"),
+    (("QA", None, None, None), "<QuerySet [<JobPost: intel QA>, <JobPost: Red Hat QA>]>"),
+    ((None, None, "STUDENT", None), "<QuerySet [<JobPost: intel sw for students>, <JobPost: Red Hat QA>]>"),
+    ])
+@pytest.mark.django_db
+def testGetSearchResults(n, expected):
+    free_search = (None, None, None, None)
+    # Free search when the courent DB is empty
+    assert str(JobPost.GetSearchResults(*free_search)) == "no relevant jobs for you"
+    # Create new DATA BASE
+    createNewDB()
+
+    # Free search when the courent DB is not empty - after created DB
+    assert str(JobPost.GetSearchResults(*free_search)) == str(JobPost.objects.all())
+    # Check few cases
+    # Check assert (realFunctionResult, expected)
+    assert str(JobPost.GetSearchResults(*n)) == expected
+
+    # Remove all the objects from DB
+    deleteAllobjects()
+    # Check that all the new objects are removed
+    checkAllObjectsAreDeleted()
+
+
+def checkAllObjectsAreDeleted():
+    empty = 0
+    assert JobPost.objects.all().count() == empty
+    assert JobType.objects.all().count() == empty
+    assert JobCity.objects.all().count() == empty
+    assert Company.objects.all().count() == empty
+
+
+def deleteAllobjects():
+    JobPost.objects.all().delete()
+    JobType.objects.all().delete()
+    JobCity.objects.all().delete()
+    Company.objects.all().delete()
+
+
+def createNewDB():
```

Comment:
The function name is incorrect - you're not creating a DB
You're initialize the DB with data